<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="settingBtn">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>10</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Image/settings.png</normaloff>:/Image/settings.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="shopBtn">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>75</width>
     <height>75</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Image/shop.png</normaloff>:/Image/shop.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>75</width>
     <height>75</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>570</y>
     <width>591</width>
     <height>103</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="beltBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Image/belt_button.png</normaloff>:/Image/belt_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="minerBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Image/miner_button.png</normaloff>:/Image/miner_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="cutterBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Image/cutter_button.png</normaloff>:/Image/cutter_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="spadeBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Image/spade.png</normaloff>:/Image/spade.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="trashBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="resource.qrc">
        <normaloff>:/Image/trash_button.png</normaloff>:/Image/trash_button.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>75</width>
        <height>75</height>
       </size>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
